hips/draw/simple.py

```diff
@@ -1,5 +1,158 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
+import os
+from pathlib import Path
+
+import healpy as hp
+import numpy as np
+from astropy.coordinates import SkyCoord
+from astropy.wcs import WCS
+from skimage import transform as tf
+from typing import List
+
+from ..tiles import HipsSurveyProperties, HipsTile, HipsTileMeta
+from ..utils import WCSGeometry, compute_healpix_pixel_indices, boundaries
 
 __all__ = [
+    'draw_sky_image',
+    'make_sky_image',
 ]
+
+
+def draw_sky_image(geometry: WCSGeometry, tiles: List[HipsTile]) -> np.ndarray:
+    """Draw sky image using the simple and quick method.
+
+    Parameters
+    ----------
+    geometry : 'WCSGeometry'
+        An object of WCSGeometry
+    tiles : List[HipsTile]
+        A list of HipsTile
+
+    Returns
+    -------
+    np.ndarray
+        Returns a numpy array containing all HiPS tiles projected onto it
+    """
+    # TODO: copy over (and clean up a little) the drawing code you already have in notebooks.
+    # Suggestion: don't debug the code now, just put it here.
+    # First implement simple tile caching so that you can draw without needing to make web queries.
+
+    # Probably this function should just loop over tiles and call a `_draw_sky_image_one_tile`
+    # helper function, and sum up the results.
+    all_sky = np.zeros(geometry.shape)
+    for tile in tiles:
+        nside = hp.order2nside(tile.meta.order)
```

Comment:
Here you're already starting to do computations that are specific to this one tile.
Those should be moved to the `_draw_sky_image_one_tile` function called below, no?